apps/bmd/package.json

```diff
@@ -96,7 +96,6 @@
     "fetch-mock": "^9.5.0",
     "history": "^4.10.1",
     "http-proxy-middleware": "^1.0.0",
-    "js-sha256": "^0.9.0",
```

Comment:
don't we need SHA256 in BMD to compute and show the election hash, since there's no backend?

Reply:
We delegate that to `safeParseElectionDefinition` from `@votingworks/types`, which means BMD doesn't _directly_ depend on this package, so it's safe to remove it.